src/Arbitrable.ts

```diff
@@ -2,7 +2,7 @@ import { Evidence } from '../types/schema'
 import { EvidenceSubmitted } from '../types/templates/Arbitrable/Arbitrable'
 
 export function handleEvidenceSubmitted(event: EvidenceSubmitted): void {
-  let id = event.transaction.hash.toHex() + event.logIndex.toHex()
+  let id = event.transaction.hash.toHex() + event.logIndex.toHex().slice(2).padStart(4, '0')
```

Comment:
What's the reason for this fix? Should we do it in the rest of the places we are generating an id using the tx hash and the log index?

Reply:
@fabriziovigevani asked for it, maybe him or @rperez89 can provide more context. It seems there was an ordering issue in the frontend (which I don't fully understand)

Reply:
Yes, the reason is that we need to have another field to order the evidences by, in the case that they were created at the same time.

Reply:
@fabriziovigevani 👇 
> Should we do it in the rest of the places we are generating an id using the tx hash and the log index?

Reply:
We have concluded in the standup that we can just slice the tx hash in the frontend and use the resulting `logIndex` for comparison. So this change is not needed.

Reply:
Ok, closing it then